node.go

```diff
@@ -73,7 +79,7 @@ func (n *Node) Close(optArgs ...CloseOpts) error {
 		}
 	}
 
-	n.refreshTicker.Stop()
```

Comment:
I think you should keep the call to `Stop` as otherwise the ticker will continue ticking.